frontends/concrete-python/Makefile

```diff
@@ -45,7 +45,7 @@ pytest-default:
 	export LD_PRELOAD=$(RUNTIME_LIBRARY)
 	export PYTHONPATH=$(BINDINGS_DIRECTORY)
 	pytest tests -svv -n auto \
-		--cov=concrete \
+		--cov=concrete.fhe \
```

Comment:
expected changes in the coverage? @umut-sahin

Reply:
Yes, bindings were tried to be covered which is not the goal of `concrete-python` tests. Our usage of the API is tested by our usage of the bindings and our own coverage.